react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+export default class Accordion extends React.Component {
+  toggleContent(event) {
+    const accordionItems = document.querySelectorAll('.label');
+    const openAccordionItem = document.querySelector('.open');
+
```

Comment:
In React apps, we generally do not modify the DOM directly. Instead, try having an initial state that indicates which topic is expanded

```js
this.state = {
  expandedTopic: null
};
```

We set it to `null` initially to indicate "none". Then, when a use a click listener method to update our state based on something that identified the `event.target`, such as the topic's title or index in the array.